nofos/guides/views.py

```diff
@@ -136,6 +138,71 @@ class ContentGuideEditView(GroupAccessObjectMixin, DetailView):
     def get_success_url(self):
         return reverse_lazy("guides:guide_edit", args=[self.object.pk])
 
+    @transaction.atomic
+    def post(self, request, *args, **kwargs):
+        """
+        Handle POST requests for saving subsection selections.
+        """
+        self.object = self.get_object()
+
+        try:
+            data = json.loads(request.body)
+            subsections = data.get("subsections", {})
+
+            updated_count = 0
+            failed_subsection_ids = []
+            for subsection_id, is_checked in subsections.items():
+                try:
+                    subsection = ContentGuideSubsection.objects.get(
+                        id=subsection_id, section__content_guide=self.object
+                    )
+
+                    new_comparison_type = "body" if is_checked else "none"
+
+                    if subsection.comparison_type != new_comparison_type:
+                        subsection.comparison_type = new_comparison_type
+                        subsection.save()
+                        updated_count += 1
+
+                except Exception as e:
+                    failed_subsection_ids.append(subsection_id)
+                    continue
+
+            if len(subsections) == len(failed_subsection_ids):
+                message = f"All comparison selections failed to update."
+                status = "fail"
+            elif len(failed_subsection_ids) > 0:
+                message = f"Some comparison selections failed to update: {",".join(failed_subsection_ids)}"
+                status = "partial"
```

Comment:
This is really nice. Amazing to get this level of feedback back from an Ajax response (if you know where to find it).

<img width="793" height="116" alt="Screenshot 2025-08-05 at 20 02 02" src="https://github.com/user-attachments/assets/0dc74bac-cd90-479c-9376-0be65515bc6b" />